test/e2e/run_passwd_test.go

```diff
@@ -137,4 +137,16 @@ USER 1000`, ALPINE)
 		Expect(run).Should(Exit(0))
 		Expect(run.OutputToString()).NotTo((ContainSubstring("1234:1234")))
 	})
+
+	It("podman run --passwd-entry flag", func() {
+		run := podmanTest.Podman([]string{"run", "--user", "1234:1234", "--passwd-entry=FOO", ALPINE, "cat", "/etc/passwd"})
+		run.WaitWithDefaultTimeout()
+		Expect(run).Should(Exit(0))
+		Expect(run.OutputToString()).To(ContainSubstring("FOO"))
```

Comment:
Can we test that the entry is in /etc/passwd too?

Reply:
we are testing that.  The container runs `cat /etc/passwd` and we expect to find `FOO`, not the UID or GID.

I'll make the test clearer and add a comment.